extras/catch_amalgamated.cpp

```diff
@@ -6,7 +6,7 @@
 // SPDX-License-Identifier: BSL-1.0
 
 //  Catch v3.2.0
-//  Generated: 2022-11-16 19:30:16.114602
```

Comment:
[No touching amalgamated files in PRs](https://github.com/catchorg/Catch2/blob/devel/docs/contributing.md#using-github)